CHANGELOG.asciidoc

```diff
@@ -0,0 +1,667 @@
+
+[[release-notes-2.16.0]]
+=== 2.16.0 - 2019/8/26
+
+[float]
+==== Features
+* feat(memcached): instrument memcached v2.2.0 and above {pull}1144[#1144]
+* feat(config): add configFile config option {pull}1303[#1303]
+
+[float]
+==== Bug fixes
+* fix: bug where spans sometimes wouldn't have stack traces {pull}1299[#1299]
+* fix(async\_hooks): properly update sync flag {pull}1306[#1306]
+* fix: change agent active status log message to debug level {pull}1300[#1300]
+
+[[release-notes-2.15.0]]
+=== 2.15.0 - 2019/8/15
+
+[float]
+==== Features
+* feat(express-graphql): add support for v0.9 {pull}1255[#1255]
+* feat(metrics): add metricsLimit option {pull}1273[#1273]
+
+[[release-notes-2.14.0]]
+=== 2.14.0 - 2019/8/12
+
+[float]
+==== Features
+* feat(hapi): support new @hapi/hapi module {pull}1246[#1246]
+* feat: allow agent.clearPatches to be called with array of names {pull}1262[#1262]
+
+[float]
+==== Bug fixes
+* fix: be less chatty if span stack traces cannot be parsed {pull}1274[#1274]
+* perf: use for-of instead of forEach {pull}1275[#1275]
+
+[[release-notes-2.13.0]]
+=== 2.13.0 - 2019/7/30
+
+[float]
+==== Bug fixes
+* fix: standardize user-agent header {pull}1238[#1238]
+
+[float]
+==== Features
+* feat: add support for APM Agent Configuration via Kibana {pull}1197[#1197]
+* feat(metrics): breakdown graphs {pull}1219[#1219]
+* feat(config): default serviceVersion to package version {pull}1237[#1237]
+
+[[release-notes-2.12.1]]
+=== 2.12.1 - 2019/7/7
+
+[float]
+==== Bug fixes
+* fix(knex): abort early on unsupported version of knex {pull}1189[#1189]
+
+[[release-notes-2.12.0]]
+=== 2.12.0 - 2019/7/2
+
+[float]
+==== Features
+* feat(metrics): add runtime metrics {pull}1021[#1021]
+* feat(config): add environment option {pull}1106[#1106]
+
+[[release-notes-2.11.6]]
+=== 2.11.6 - 2019/6/11
+
+[float]
+==== Bug fixes
+* fix(express): don't swallow error handling middleware {pull}1111[#1111]
+
+[[release-notes-2.11.5]]
+=== 2.11.5 - 2019/5/27
+
+[float]
+==== Bug fixes
+* fix(metrics): report correct CPU usage on Linux {pull}1092[#1092]
+* fix(express): improve names for routes added via app.use() {pull}1013[#1013]
+
+[[release-notes-2.11.4]]
+=== 2.11.4 - 2019/5/27
+
+[float]
+==== Bug fixes
+* fix: don't add traceparent header to signed AWS requests {pull}1089[#1089]
+
+[[release-notes-2.11.3]]
+=== 2.11.3 - 2019/5/22
+
+[float]
+==== Bug fixes
+* fix(span): use correct logger location {pull}1081[#1081]
+
+[[release-notes-2.11.2]]
+=== 2.11.2 - 2019/5/21
+
+[float]
+==== Bug fixes
+* fix: url.parse expects req.url not req {pull}1074[#1074]
+* fix(express-slash): expose express handle properties {pull}1070[#1070]
+
+[[release-notes-2.11.1]]
+=== 2.11.1 - 2019/5/10
+
+[float]
+==== Bug fixes
+* fix(instrumentation): explicitly use `require` {pull}1059[#1059]
+* chore: add Node.js 12 to package.json engines field {pull}1057[#1057]
+
+[[release-notes-2.11.0]]
+=== 2.11.0 - 2019/5/3
```

Comment:
We should probably leave the commit grouping headers in, even when there's only one type, just to be clear.